source/SpinalHDL/Getting Started/VSCodium.rst

```diff
@@ -4,12 +4,13 @@ Using Spinal from VSCodium
 ==========================
 
 .. note::
-   VSCode works the same way as VSCodium.
+    `VSCodium <https://vscodium.com/>`_ is the open source Microsoft Visual Studio application, but without the telemetry included in Microsoft's downloadable version.  
 
-As a one-time setup task, in the extensions options, install "Scala (Metals)".
+As a one-time setup task, go to view->extensions search for "Scala" and install the "Scala (Metals)" extemsion. `WebPage <https://marketplace.visualstudio.com/items?itemName=scalameta.metals>`_
 
-Open the workspace: ``File`` > ``Open Folder...`` and open the folder you have
-downloaded earlier in :ref:`template`.
+Open the workspace: ``File`` > ``Open Folder...`` and open the folder you have downloaded earlier in :ref:`template`.
+
+The other way to start it, is to cd into the spinaltemplatesbt directory and type ``codium .``
```

Comment:
If one did follow the previous commands it would actually be in "MySpinalProject" - maybe just skip this as it's bound to break?

Reply:
What I did on this one, is that I went back to my fork of this repository and removed the word "spinaltemplatesbt " and replaced it with the word "appropriate". 

i, and presumably some others, find that more natural to do. 

I am not sure if that change got propagated to you or not.  I am still a bit new to this interface.  And it has been a long time since I did restructured text stuff.  I will read the manual more carefully next time I do an edit of a path or url.

Reply:
by the looks of it, it worked out